.github/workflows/build.yml

```diff
@@ -6,9 +6,9 @@ name: Gating
 # events but only for the main branch
 on:
   push:
-    branches: [ '*' ]
+    branches: [ maint-1.3 ]
   pull_request:
-    branches: [ main, maint-1.3 ]
+    branches: [ maint-1.3 ]
```

Comment:
I think this should be reverted to be as in the original file from `main`.

Reply:
yes!